test/multifile_samples/macros/instrumentationMetadata.json.expected

```diff
@@ -274,11 +274,13 @@
         "154:37:2",
         "72:21:0"
       ],
+      "annotationText": "OwnershipTransferred(address indexed ",
```

Comment:
This also looks like a fragment

Reply:
Good catch. The code wasn't correctly computing the original text for macro instantiations. Fixed in [fcd2351](https://github.com/ConsenSys/scribble/pull/179/commits/fcd235144e433fcac6ec0791439313e222dcf60f).